.github/workflows/build-and-test.yaml

```diff
@@ -0,0 +1,56 @@
+name: Build and Test
+on:
```

Comment:
As in [era-compiler-vyper/pull/11](https://github.com/matter-labs/era-compiler-vyper/pull/11/files), 

> I’d also suggest a
 workflow_dispatch:
entry, like that at [compiler-llvm/.github/workflows/compiler-integration-tests.yml#L7](https://github.com/matter-labs/compiler-llvm/blob/dev-15/.github/workflows/compiler-integration-tests.yml#L7), to get a Run Workflow button like that at [compiler-integration-tests](https://github.com/matter-labs/compiler-llvm/actions/workflows/compiler-integration-tests.yml).

Reply:
Sure. It's already in place after `push` and `pull_request` targets.

Reply:
Duh, thanks.